arduino/examples/ESP32IOBoard/ESP32IOBoard.ino

```diff
@@ -137,6 +126,10 @@ string dummystring("abcdef");
 /// layout. The argument of offset zero is ignored and will be removed later.
 static constexpr openlcb::ConfigDef cfg(0);
 
+#if defined(USE_WIFI)
+Esp32WiFiManager wifiMgr(ssid, password, openmrn.stack(), cfg.seg().wifi());
```

Comment:
what is the desired style for global variables for you?
After this change there will be both lower_case_underscore and camelCase in the same file.

Reply:
Either is fine, I'll update the code to match existing standards.